ioctl/cmd/account/accountactions.go

```diff
@@ -73,130 +76,51 @@ func accountActions(args []string) error {
 	if err != nil {
 		return output.NewError(output.AddressError, "failed to get address", err)
 	}
-
-	conn, err := util.ConnectToEndpoint(config.ReadConfig.SecureConnect && !config.Insecure)
-	if err != nil {
-		return output.NewError(output.NetworkError, "failed to connect to endpoint", err)
+	reqData := map[string]string{
+		"address": addr,
+		"offset":  fmt.Sprint(skip),
 	}
-	defer conn.Close()
-	cli := iotexapi.NewAPIServiceClient(conn)
-	ctx := context.Background()
-
-	jwtMD, err := util.JwtAuth()
-	if err == nil {
-		ctx = metautils.NiceMD(jwtMD).ToOutgoing(ctx)
-	}
-
-	requestGetAccount := iotexapi.GetAccountRequest{
-		Address: addr,
-	}
-	accountResponse, err := cli.GetAccount(ctx, &requestGetAccount)
+	jsonData, err := json.Marshal(reqData)
 	if err != nil {
-		return output.NewError(output.APIError, "failed to get account", err)
-	}
-	numActions := accountResponse.AccountMeta.GetNumActions()
-	fmt.Println("Total:", numActions)
-	requestGetAction := iotexapi.GetActionsRequest{
-		Lookup: &iotexapi.GetActionsRequest_ByAddr{
-			ByAddr: &iotexapi.GetActionsByAddressRequest{
-				Address: addr,
-				Start:   numActions - *countLimit - skip,
-				Count:   *countLimit,
-			},
-		},
+		return output.NewError(output.ConvertError, "failed to pack in json", nil)
 	}
-	response, err := cli.GetActions(ctx, &requestGetAction)
+	resp, err := http.Post("https://iotexscout.io/apiproxy/api.ActionsService.GetAllActionsByAddress", "application/json",
```

Comment:
URL has moved to `https://iotex-analyser-api-mainnet.chainanalytics.org/api.ActionsService.GetActionsByAddress`

Reply:
Better to make it an ioctl config.